src/lib/server/services/content.ts

```diff
@@ -268,6 +270,27 @@ export const getPostStatistics = async (db: DB) => {
 };
 export type PostStatistics = Awaited<ReturnType<typeof getPostStatistics>>;
 
+export const createCommunityReport = async (
+    db: DB,
+    newPostCommunityReport: NewPostReport
```

Comment:
Can you change the name of this function to `createChannelReport` and the `newPostCommunityReport` param to `newChannelPostReport` just for consistency throughout the site.

Also, the type for the param should be `NewChannelPostReport` and it returns a `ChannelPostReport`.